### ISSUE TITLE
Error?

### ISSUE THREAD

**Issue by oh111** [warnakey/delete-reddit-posts#3]
Everything works till the Password is put in. Then this error appears: 
   Attempting to login to Reddit with the provided Username and Password.
   Successfully logged in to Reddit with the provided Username and Password.
   This reddit account is using the old version of Reddit.
   Unable to find the profile page link - now the program will crash.
   This account has no posts to delete. Try creating some posts on Reddit first before running this 
   program.
   Program complete. Closing out of Puppeteer and exiting the program.

**Comment by warnakey** [comment]
You need to enter your username and password inside the terminal, NOT the
browser! Please try that and let me know if it works.

**Comment by oh111** [comment]
Yup-- I type it in Terminal-- then it gets autotyped in Chrome and ultimately, that is the error I get!

**Comment by warnakey** [comment]
Can you provide any more details? Browser? Type of Reddit account? This may
be an edge case I didn't account for.

**Comment by warnakey** [comment]
Hello @oh111 

I've discovered this happens on accounts with 2 factor authentication. If that was the case for you, I have updated the program to account for 2FA and everything should work now. Thanks
